lib/ex_aws/ses.ex

```diff
@@ -0,0 +1,159 @@
+
+defmodule ExAws.SES do
+  import ExAws.Utils, only: [camelize_key: 1, camelize_keys: 1]
+
+  @moduledoc """
+  Operations on AWS SES
+
+  http://docs.aws.amazon.com/ses/latest/APIReference/Welcome.html
+  """
+
+  @doc "Verifies an email address"
+  @spec verify_email_identity(email :: binary) :: ExAws.Operation.Query.t
+  def verify_email_identity(email) do
+    request(:verify_email_identity, %{"EmailAddress" => email})
+  end
+
+  @doc "Fetch identities verification status and token (for domains)"
+  @spec identity_verification_attributes([binary]) :: ExAws.Operation.Query.t
+  def identity_verification_attributes(identities) when is_list(identities) do
+    params = format_member_attribute({:identities, identities})
+    request(:get_identity_verification_attributes, params)
+  end
+
+  @doc "Fetch configuration sets associated with AWS account"
+  @spec configuration_sets(list) :: ExAws.Operation.t
+  def configuration_sets(opts \\ []) do
+    params = build_opts(opts, [:max_items, :next_token])
+    request(:list_configuration_sets, params)
+  end
+
+
+  ## Emails
+  ######################
+  @type message :: %{
+    body: %{html: %{data: binary, charset: binary}, text: %{data: binary, charset: binary}},
+    subject: %{data: binary, charset: binary}
+  }
+  @type destination :: %{to: [binary], bcc: [binary], bcc: [binary]}
```

Comment:
@lessless Do note that I shortened the API here a bit from `to_addresses` and so on to just `to`. They're all addresses, I didn't feel like suffixing them added value.